src/Clock.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends React.Component<Props, State> {
+  state: Readonly<State> = {
+    today: new Date(),
+  };
+
+  timeId = 0;
+
+  componentDidMount() {
+    this.timeId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+    }, 1000);
+  }
+
```

Comment:
you miss, console info when it mount, and console debug when update. add update component logic